maven/bnd-export-maven-plugin/pom.xml

```diff
@@ -52,6 +51,33 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.10</version>
```

Comment:
The version should be in the parent pom. We try to control this from the parent pom.